lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.js

```diff
@@ -111,6 +126,16 @@ testRule({
 			fixed: 'a { border-width: 1px 1px 1px 1px; }',
 			message: messages.expected('border-width'),
 		},
+		{
+			code: 'div { top: 1px; right: 2px; bottom: 3px; left: 4px; }',
+			fixed: 'div { inset: 1px 2px 3px 4px; }',
+			message: messages.expected('inset'),
+		},
+		{
+			code: 'div { top: 0; right: 0; bottom: 0; left: 0; }',
+			fixed: 'div { inset: 0 0 0 0; }',
```

Comment:
[suggest] Also here.

```suggestion
			code: 'a { top: 1px; right: 2px; bottom: 3px; left: 4px; }',
			fixed: 'a { inset: 1px 2px 3px 4px; }',
			message: messages.expected('inset'),
		},
		{
			code: 'a { top: 0; right: 0; bottom: 0; left: 0; }',
			fixed: 'a { inset: 0 0 0 0; }',
```